### PR TITLE
Added Radio Buttons

### PR DESCRIPTION
**Pull request by ayushgoswamiabr**
I Have Added Radio Buttons

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,35 @@
...
+            </span>
+            <span>
+              <input type="radio" value="SendGrid" id="SendGrid" name="mail" onChange={()=>{setservice('SendGrid')}} checked={service==='SendGrid'}/>
+              <label for="SendGrid">SendGrid</label>
+            </span>
+      </form>
+      {service && <h3>You have Chosen {service}</h3>}
+      {service && <button onClick={()=>{setservice('')}}>Clear Selection</button>}
+    </div>
+    </>
```
Comment (ANISH0309) [inline src/App.js:32]: Please put this in a separate file under the components folder. "App.js" will get cluttered in long run. :)

File: src/index.css [hunk 2]
```diff
@@ -2,3 +2,32 @@
...
+    color: #fff;
+}
+.radio-form{
+    display: flex;
+    flex-direction: column;
+}
+
+.radio-form input{
+    margin-right: 3px;
+}
```
Comment (ANISH0309) [inline src/index.css:33]: We are doing this project in Tailwind CSS. Kindly remove the plain CSS and make the necessary changes.

**Review by ANISH0309 — CHANGES_REQUESTED** [review]

**Comment by ayushgoswamiabr** [comment]
Ok I will do the necessary changes

**Comment by ayushgoswamiabr** [comment]
I have Made the Radio.js component and I have removed custom css and Implemented Tailwind css Please Tell if any Design Changes are needed

**Review by ANISH0309 — CHANGES_REQUESTED** [review]
Hey, great going. Can you please refer to our [notion guide](https://grass-text-1a7.notion.site/Maildart-9dec4c315dcf407fa73d06a3023edc6c) for the design and try to make it look similar.

**Comment by ayushgoswamiabr** [comment]
Yeah Sure

**Comment by ayushgoswamiabr** [comment]
I have done Styling according to the notion guide

File: src/Components/Radio.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+const Radio = () => {
+    const [service, setservice] = useState('')
+    return (
+  <div className="bg-black text-white flex p-1">
+    <div className="flex-1">
+      <form >
+        <span className="p-4">
+              <input className="form-radio text-indigo-600" type="radio" value="Listmonk" id="Listmonk" name="mail" onChange={()=>{setservice('Listmonk')}} checked={service==='Listmonk'}/>
+              <label for="listmonk">Listmonk</label>
```
Comment (ANISH0309) [inline src/Components/Radio.js:11]: Add some spacing between all radio buttons and their label

**Review by ANISH0309 — CHANGES_REQUESTED** [review]
Also by convention, the "components" folder is always in lowercase, change that also.

File: src/App.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import Radio from './Components/Radio'
+import Radio from './components/Radio'
```
Comment (ANISH0309) [inline src/App.js:1]: You also have to change the casing of the "components" folder under the "src" folder, then only this line will work.

File: src/Components/Radio.js [hunk 5]
```diff
@@ -7,19 +7,19 @@ const Radio = () => {
     <div className="flex-1">
       <form >
         <span className="p-4">
-              <input className="form-radio text-indigo-600" type="radio" value="Listmonk" id="Listmonk" name="mail" onChange={()=>{setservice('Listmonk')}} checked={service==='Listmonk'}/>
+              <input className="m-0.5" type="radio" value="Listmonk" id="Listmonk" name="mail" onChange={()=>{setservice('Listmonk')}} checked={service==='Listmonk'}/>
```
Comment (ANISH0309) [inline src/Components/Radio.js:10]: The spacing between the radio button and the label is negligible. A little more spacing between them would make the UI look even better :)

**Review by ANISH0309 — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 4 above)
Comment (ayushgoswamiabr) [reply src/App.js:1]: I have done that

Comment (ANISH0309) [reply src/App.js:1]: Screenshot 2021-10-05 at 1 42 36 AM


I can't see any changes being made here.

Comment (ayushgoswamiabr) [reply src/App.js:1]: Git wasn't detecting rename when I renamed directly through vs code but It was detected when I renamed the folder from terminal

**Review by ANISH0309 — APPROVED** [review]
Well done @ayushgoswamiabr!

**Pull request merged**